File: Prereq-OOP/src/zoo/Animal.java
package zoo;

// Superclass / Parent class
public class Animal {

    int age, weightInLbs;
    String gender;

    public Animal(int age, String gender, int weightInLbs) {
        this.age = age;
        this.gender = gender;
        this.weightInLbs = weightInLbs;
    }

    public void eat() {
        System.out.println("eating...");
    }

    public void sleep() {
        System.out.println("sleeping...");
    }
}

File: Python/Zoo/animal.py
class Animal:

    # Constructor
    def __init__(self, age, gender, weightInLbs):
        self.age = age
        self.gender = gender
        self.weightInLbs = weightInLbs

    def eat(self):
        print('eating...')

    def sleep(self):
        print('sleeping...')